crawler/src/main/java/com/movie/recommender/crawler/Main.java

```diff
@@ -22,6 +26,7 @@ private static void initializeMovieData(ApplicationContext context) {
             return;
         }
 
+        // Initialize movie data
```

Comment:
in this method, remove the `DATASETPATH` environment variable and create a configuration property
something like this
```
dataset:
  path: /path/to/dataset/file.json
```

do not read this property here
instead, add it as a configuration field in `MovieDataInitializer`
```
@Value("${dataset.path}")
private String datasetPath;
```